i18n/en.json

```diff
@@ -2952,7 +2948,7 @@
   },
   {
     "id": "app.plugin.get_cluster_plugin_statuses.app_error",
-    "translation": ""
+    "translation": "Unable to get cluster plugin statuses."
```

Comment:
`Unable to get plugin statuses from the cluster.` --> or keep if it's about "cluster plugin" statuses